.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should be removed from the repository. You can keep it locally, but it should not be here.

Use:
```
git rm --cached -r vscode``` 
to do that.